spec/integration/soapy_cake/admin_track_spec.rb

```diff
@@ -1,18 +1,8 @@
 require 'spec_helper'
 
 RSpec.describe SoapyCake::AdminTrack do
-  describe '#decrypt_affiliate_link' do
-    let(:service) { :track }
-    let(:cake_method) { :decrypt_affiliate_link }
-    let(:method) { :decrypt_affiliate_link }
-    let(:cake_opts) { { a: 1 } }
-    let(:opts) { nil }
-
-    it_behaves_like 'a cake admin method'
-  end
-
```

Comment:
This was duplicated in `spec/lib/soapy_cake/admin_track_spec.rb` (and not really an integration spec anyway), so I took it out.